.github/workflows/release_workflow.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI for releasing tagged versions
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Is it a regex? Doesn't "." mean any character here?

Reply:
According to https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags it is not a regular expression, it is a "filter pattern", so the syntax is a bit different. In this case, `.` means a "dot", not "any character".

You made me doubt! :-D